libs/pika/execution_base/include/pika/execution_base/any_sender.hpp

```diff
@@ -815,6 +838,7 @@ namespace pika::execution::experimental {
             PIKA_UNREACHABLE;
         }
 
+        /// \brief Assign \p sender to the \ref unique_any_sender.
```

Comment:
just out of curiosity, is it anyhow different from `operator=(Sender)`?

Reply:
No, it's essentially the same. I think it's not super essential functionality, but I modeled it after how `unique_ptr::reset` behaves (https://en.cppreference.com/w/cpp/memory/unique_ptr/reset). A plain `reset()` is the main use case to empty it, but one can also do `reset(sender)`.